src/Sylius/Bundle/GridBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\GridBundle\DependencyInjection;
+
+use Sylius\Bundle\ResourceBundle\SyliusResourceBundle;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('sylius_grid');
+
+        $this->addGridsSection($rootNode);
+
+        return $treeBuilder;
+    }
+
+    /**
+     * Adds `grids` section.
+     *
+     * @param ArrayNodeDefinition $node
+     */
+    private function addGridsSection(ArrayNodeDefinition $node)
+    {
+        $node
+            ->children()
+                ->arrayNode('grids')
+                    ->useAttributeAsKey('code')
+                    ->prototype('array')
+                        ->children()
+                            ->arrayNode('driver')
+                                ->children()
+                                    ->scalarNode('name')->defaultValue('doctrine/orm')->end()
+                                    ->arrayNode('options')
+                                        ->prototype('variable')->end()
+                                        ->defaultValue(array())
+                                    ->end()
+                                ->end()
+                            ->end()
+                            ->arrayNode('sorting')
+                                ->prototype('scalar')->end()
+                            ->end()
+                            ->arrayNode('fields')
```

Comment:
That's up for discussion, in theory yes - when consuming via API, fields do not matter. BUT, there is sorting, where they could matter. So the question is where to configure sorting logic, in ``fields`` or separately? Probably the latter is cleaner and solves the problem. I will leave it as is right now.